RELEASE-NOTES.md

```diff
@@ -1,3 +1,32 @@
+### 0.11.0
+#### Compatibility
+- golem-messages: 2.24.0
+- golem-smart-contracts-interface: 1.6.1
+
+#### Endpoints and uses cases in general
+- Bugfix: Change concent response from ServiceRefused to Http404 (#1051).
+- Bugfix: Validate protocol version (#1057).
+- Bugfix: Fix non nesting atomic to verify transaction nesting in runtime instead of on startup (#1053).
+- Bugfix: Wrong jpeg handling in verification use case (#1046).
```

Comment:
This could be more specific. What was wrong with JPEG handling? The changelog should say that.

Reply:
Concent fail on assert:
```
assert output_format in BlenderSubtaskDefinition.OutputFormat.__members__.keys()
```
We didn't have `JPEG` specify from what I understand, only `JPG`. We can change description so that we add or fix `JPEG` handling in verification?